src/map.cpp

```diff
@@ -1482,6 +1485,7 @@ void ServerMap::finishBlockMake(BlockMakeData *data,
 		NOTE: Will be saved later.
 	*/
 	//save(MOD_STATE_WRITE_AT_UNLOAD);
+	m_chunks_in_progress.erase(bpmin);
```

Comment:
which lock protect this?
is it the env mutex?

Reply:
All use of this is under the `Server::m_env_mutex` lock.